src/libs/WorkflowUtils.ts

```diff
@@ -1,14 +1,17 @@
 import lodashMapKeys from 'lodash/mapKeys';
-import type {Approver, Member} from '@src/types/onyx/ApprovalWorkflow';
+import type {ValueOf} from 'type-fest';
+import CONST from '@src/CONST';
+import type {ApprovalWorkflowOnyx, Approver, Member} from '@src/types/onyx/ApprovalWorkflow';
 import type ApprovalWorkflow from '@src/types/onyx/ApprovalWorkflow';
 import type {PersonalDetailsList} from '@src/types/onyx/PersonalDetails';
 import type {PolicyEmployeeList} from '@src/types/onyx/PolicyEmployee';
 
-const EMPTY_APPROVAL_WORKFLOW: ApprovalWorkflow = {
+const EMPTY_APPROVAL_WORKFLOW: ApprovalWorkflowOnyx = {
```

Comment:
@blazejkustra  NAB: These places are my suggestions because I think these words will be more familiar to the current codebase, you can apply them If you agree with me. 
```suggestion
const INITIAL_APPROVAL_WORKFLOW: ApprovalWorkflowOnyx = {
```

Reply:
I like it! Changed @DylanDylann